tool/fakerelay/main.go

```diff
@@ -15,11 +15,16 @@ import (
 )
 
 var (
+	// Number of requests received by fakerelay instance
 	requestCount = expvar.NewInt("requests")
 
+	// string representation of first request received by fakerelay instance
```

Comment:
This should be above the `firstRequest`, not above the group.

I don't find the 3 new comments adding much value here, they basically repeat the variable name plus "received by fakerelay" -- but the context that this is fakerelay should be clear, it couldn't be related to anything else, could it?

Reply:
removed the comments